packages/block-editor/src/components/block-list/block-list-item.native.js

```diff
@@ -106,7 +131,12 @@ export class BlockListItem extends Component {
 		return (
 			<ReadableContentView
 				align={ blockAlignment }
-				style={ readableContentViewStyle }
+				style={ [
+					readableContentViewStyle,
+					parentWidth && {
+						maxWidth: parentWidth + 2 * marginHorizontal,
+					},
```

Comment:
I think this is breaking cases like `Group`  (Full-width) > `Cover` no alignment selected.

Before | After
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/100589260-2c809c80-32f3-11eb-8e47-95794439e7a4.png" width="250" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/100589299-36a29b00-32f3-11eb-941b-9c3cb09f14b7.png" width="250" /></kbd> 



<details><summary>HTML to test</summary>

```
<!-- wp:group {"align":"full"} -->
<div class="wp-block-group alignfull"><div class="wp-block-group__inner-container"><!-- wp:cover {"customOverlayColor":"#cd2653","align":"center"} -->
<div class="wp-block-cover aligncenter has-background-dim" style="background-color:#cd2653"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…"} -->
<p class="has-text-align-center"></p>
<!-- /wp:paragraph --></div></div>
<!-- /wp:cover --></div></div>
<!-- /wp:group -->
```
</details>

Reply:
I think it was the requirement - children should take the whole available space within _full width parent_, however it looks like it's against the current web behavior. Wdyt @iamthomasbishop

Reply:
> I'm hoping that we can force children within to utilize all of the available space, but it looks like we're probably applying the standard 580px container within them

Or that [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/14891#issuecomment-693012465) should be applied only to children nested within column, or maybe only to `Paragraphs` 🤔 Definitely need more info @iamthomasbishop

Reply:
Yeah, it'd be good to have those clarifications =)